lib/zendesk_apps_support/assets/templates.js.erb

```diff
@@ -0,0 +1,8 @@
+<% unless templates.nil? %>
+{
+  <% templates.each do |name, fn| %>
+  <%= name %>: <%= fn %>,
```

Comment:
Well now you know I tried your code ;-). I actually programmed this thing myself, not realizing you did it already.

I used npm and `.bin/handlebars #{file}` though. But I like your approach more.